### ISSUE TITLE
build tip: "No such module 'TrustCore"

### ISSUE THREAD

**Issue by titancat** [TrustWallet/trust-wallet-ios#786]
I'm new in iOS dev.
I tried to run trust-wallet-ios code in Xcode, I got the tip: `"No such module 'TrustCore"`
After searching in Google, I found a repo [trust-core](https://github.com/TrustWallet/trust-core)
Can I fix the tip with trust-core repo? Should I compile it and put it to trust-wallet-ios folder ?
I need your help.

**Comment by vikmeup** [comment]
@titancat Make sure to run latest Xcode and then run make bootstrap

**Comment by titancat** [comment]
@vikmeup I  run code in  Xcode Version 9.4.1 (9F2000) and I make sure run `make bootstrap` successfully.

![image](https://user-images.githubusercontent.com/3294102/42025030-31df2596-7af6-11e8-8acf-5ca65f2d2333.png)

**Comment by vikmeup** [comment]
@titancat Strange, Travis works fine, same on my local machine. 

Can you try to remove derived data and clean project? And do it again

**Comment by titancat** [comment]
@vikmeup How to  remove derived data ? Can I remove trust-wallet-ios folder directly?

**Comment by steve7688** [comment]
@titancat maybe you should run pod setup before you run make bootstrap.I get this problem too.But now I can run the project perfectly.

**Comment by steve7688** [comment]
@titancat Hi  titancat   I sent you an E-mail.Check it out:)

**Comment by hillox** [comment]
@steve7688 I got the same problem. How to solve it?

**Comment by ruizhaoz1** [comment]
我碰到同样的问题。make bootstrap成功，编译时import TrustCore失败。
bogon:desktop zxl$ cd trust-wallet-ios
bogon:trust-wallet-ios zxl$ ls
CODE_OF_CONDUCT.md	Podfile			TrustTests
Dangerfile		Podfile.lock		TrustUITests
Gemfile			README.md		docs
Gemfile.lock		Trust			resources
LICENSE			Trust.xcodeproj		scripts
Makefile		Trust.xcworkspace
bogon:trust-wallet-ios zxl$ make bootstrap
Password:
Cocopods is not installed. Use sudo gem install cocoapods
Successfully installed concurrent-ruby-1.0.5
Successfully installed i18n-0.9.5
Successfully installed thread_safe-0.3.6
Successfully installed tzinfo-1.2.5
Successfully installed activesupport-4.2.10
Successfully installed nap-1.1.0
Successfully installed fuzzy_match-2.0.4
Successfully installed cocoapods-core-1.5.3
Successfully installed claide-1.0.2
Successfully installed cocoapods-deintegrate-1.0.2
Successfully installed cocoapods-downloader-1.2.1
Successfully installed cocoapods-plugins-1.0.0
Successfully installed cocoapods-search-1.0.0
Successfully installed cocoapods-stats-1.0.0
Successfully installed netrc-0.11.0
Successfully installed cocoapods-trunk-1.3.0
Successfully installed cocoapods-try-1.1.0
Successfully installed molinillo-0.6.5
Successfully installed atomos-0.1.2
Successfully installed CFPropertyList-3.0.0
Successfully installed colored2-3.1.2
Successfully installed nanaimo-0.2.6
Successfully installed xcodeproj-1.5.9
Successfully installed escape-0.0.4
Successfully installed fourflusher-2.0.1
Successfully installed gh_inspector-1.1.3
Successfully installed ruby-macho-1.2.0
Successfully installed cocoapods-1.5.3
Parsing documentation for concurrent-ruby-1.0.5
Installing ri documentation for concurrent-ruby-1.0.5
Parsing documentation for i18n-0.9.5
Installing ri documentation for i18n-0.9.5
Parsing documentation for thread_safe-0.3.6
Installing ri documentation for thread_safe-0.3.6
Parsing documentation for tzinfo-1.2.5
Installing ri documentation for tzinfo-1.2.5
Parsing documentation for activesupport-4.2.10
Installing ri documentation for activesupport-4.2.10
Parsing documentation for nap-1.1.0
Installing ri documentation for nap-1.1.0
Parsing documentation for fuzzy_match-2.0.4
Installing ri documentation for fuzzy_match-2.0.4
Parsing documentation for cocoapods-core-1.5.3
Installing ri documentation for cocoapods-core-1.5.3
Parsing documentation for claide-1.0.2
Installing ri documentation for claide-1.0.2
Parsing documentation for cocoapods-deintegrate-1.0.2
Installing ri documentation for cocoapods-deintegrate-1.0.2
Parsing documentation for cocoapods-downloader-1.2.1
Installing ri documentation for cocoapods-downloader-1.2.1
Parsing documentation for cocoapods-plugins-1.0.0
Installing ri documentation for cocoapods-plugins-1.0.0
Parsing documentation for cocoapods-search-1.0.0
Installing ri documentation for cocoapods-search-1.0.0
Parsing documentation for cocoapods-stats-1.0.0
Installing ri documentation for cocoapods-stats-1.0.0
Parsing documentation for netrc-0.11.0
Installing ri documentation for netrc-0.11.0
Parsing documentation for cocoapods-trunk-1.3.0
Installing ri documentation for cocoapods-trunk-1.3.0
Parsing documentation for cocoapods-try-1.1.0
Installing ri documentation for cocoapods-try-1.1.0
Parsing documentation for molinillo-0.6.5
Installing ri documentation for molinillo-0.6.5
Parsing documentation for atomos-0.1.2
Installing ri documentation for atomos-0.1.2
Parsing documentation for CFPropertyList-3.0.0
Installing ri documentation for CFPropertyList-3.0.0
Parsing documentation for colored2-3.1.2
Installing ri documentation for colored2-3.1.2
Parsing documentation for nanaimo-0.2.6
Installing ri documentation for nanaimo-0.2.6
Parsing documentation for xcodeproj-1.5.9
Installing ri documentation for xcodeproj-1.5.9
Parsing documentation for escape-0.0.4
Installing ri documentation for escape-0.0.4
Parsing documentation for fourflusher-2.0.1
Installing ri documentation for fourflusher-2.0.1
Parsing documentation for gh_inspector-1.1.3
Installing ri documentation for gh_inspector-1.1.3
Parsing documentation for ruby-macho-1.2.0
Installing ri documentation for ruby-macho-1.2.0
Parsing documentation for cocoapods-1.5.3
Installing ri documentation for cocoapods-1.5.3
Done installing documentation for concurrent-ruby, i18n, thread_safe, tzinfo, activesupport, nap, fuzzy_match, cocoapods-core, claide, cocoapods-deintegrate, cocoapods-downloader, cocoapods-plugins, cocoapods-search, cocoapods-stats, netrc, cocoapods-trunk, cocoapods-try, molinillo, atomos, CFPropertyList, colored2, nanaimo, xcodeproj, escape, fourflusher, gh_inspector, ruby-macho, cocoapods after 44 seconds
28 gems installed
All dependencies was installed
pod install --repo-update
Updating local specs repositories
Setting up CocoaPods master repo
  $ /usr/local/bin/git clone https://github.com/CocoaPods/Specs.git master
  --progress
  Cloning into 'master'...
  remote: Counting objects: 2325196, done.        
  remote: Compressing objects: 100% (440/440), done.        
  remote: Total 2325196 (delta 191), reused 41 (delta 37), pack-reused 2324707        
  Receiving objects: 100% (2325196/2325196), 558.58 MiB | 245.00 KiB/s, done.
  Resolving deltas: 100% (1340904/1340904), done.
  Checking out files: 100% (255780/255780), done.
Setup completed
  $ /usr/local/bin/git -C /Users/zxl/.cocoapods/repos/master fetch origin
  --progress
  remote: Counting objects: 196, done.        
  remote: Compressing objects: 100% (127/127), done.        
  remote: Total 196 (delta 123), reused 110 (delta 63), pack-reused 0        
  Receiving objects: 100% (196/196), 21.95 KiB | 48.00 KiB/s, done.
  Resolving deltas: 100% (123/123), completed with 63 local objects.
  From https://github.com/CocoaPods/Specs
     ec288596fbd..fc7d56084a8  master     -> origin/master
  $ /usr/local/bin/git -C /Users/zxl/.cocoapods/repos/master rev-parse
  --abbrev-ref HEAD
  master
  $ /usr/local/bin/git -C /Users/zxl/.cocoapods/repos/master reset --hard
  origin/master
  HEAD is now at fc7d56084a8 [Add] LayoutKit 8.0.3
Analyzing dependencies
Pre-downloading: `JSONRPCKit` from `https://github.com/bricklife/JSONRPCKit.git`, commit `50d19a4f7ec593ac5e07cffa1e11c17f1fbe347d`
Pre-downloading: `QRCodeReaderViewController` from `https://github.com/yannickl/QRCodeReaderViewController.git`, commit `80bd79cbeede842949b229f81bc6328f53701c5a`
Pre-downloading: `TrustCore` from `https://github.com/TrustWallet/trust-core`, commit `f987ee12f98c2df9d1538d86448dd8e671669350`
Pre-downloading: `TrustKeystore` from `https://github.com/TrustWallet/trust-keystore`, commit `5829d9631af1572c65108692a09d823c504aea7b`
Pre-downloading: `TrustWalletSDK` from `https://github.com/TrustWallet/TrustSDK-iOS`, commit `99eb1456c6ebf1ce6a2dca15731ff99d69f93eb0`
Pre-downloading: `TrustWeb3Provider` from `https://github.com/TrustWallet/trust-web3-provider`, commit `f4e0ebb1b8fa4812637babe85ef975d116543dfd`
Downloading dependencies
Installing APIKit (3.1.1)
Installing Alamofire (4.6.0)
Installing BigInt (3.0.1)
Installing Branch (0.24.1)
Installing Crashlytics (3.10.1)
Installing CryptoSwift (0.10.0)
Installing Eureka (4.2.0)
Installing Fabric (1.7.6)
Installing JSONRPCKit (2.0.0)
Installing KeychainSwift (10.0.0)
Installing Kingfisher (4.6.1)
Installing MBProgressHUD (1.1.0)
Installing Moya (10.0.1)
Installing PromiseKit (6.2.4)
Installing QRCodeReaderViewController (4.0.2)
Installing R.swift (4.0.0)
Installing R.swift.Library (4.0.0)
Installing Realm (3.7.1)
Installing RealmSwift (3.7.1)
Installing Result (3.2.4)
Installing SAMKeychain (1.5.3)
Installing SeedStackViewController (0.4.0)
Installing SipHash (1.2.0)
Installing StatefulViewController (3.0)
Installing SwiftLint (0.25.0)
Installing TrezorCrypto (0.0.6)
Installing TrustCore (0.0.7)
Installing TrustKeystore (0.4.1)
Installing TrustWalletSDK (0.0.1)
Installing TrustWeb3Provider (0.1.7)
Installing URLNavigator (2.0.4)
Generating Pods project
Integrating client project
Sending stats
Pod installation complete! There are 25 dependencies from the Podfile and 31 total pods installed.

**Comment by Ins0mniaIco** [comment]
I am having the exact same issue building this in xCode. Any news on how to fix this?

**Comment by alant** [comment]
tl;dr: You must reopen project .xcworkspace file(not .xcodeproj) after install your podfile.
This SO article helped resolving it for me.  https://stackoverflow.com/a/42518063/3781358

**Comment by titancat** [comment]
@alant Thanks for your advice! It works!
